Mathlib/NumberTheory/PrimeCounting.lean

```diff
@@ -102,7 +102,31 @@ theorem primeCounting'_add_le {a k : ℕ} (h0 : 0 < a) (h1 : a < k) (n : ℕ) :
 theorem zeroth_prime_eq_two : nth Prime 0 = 2 := nth_count prime_two
 
 /-- The `n`th prime is greater or equal to `n + 2`. -/
-lemma add_two_le_nth_prime (n : ℕ) : n + 2 ≤ nth Prime n :=
+theorem add_two_le_nth_prime (n : ℕ) : n + 2 ≤ nth Prime n :=
   zeroth_prime_eq_two ▸ (nth_strictMono infinite_setOf_prime).add_le_nat n 0
 
+@[simp]
+theorem range_primeCounting' : Set.range π' = Set.univ := by
```

Comment:
It's probably more idiomatic to state that this is a surjective function instead.

Reply:
makes sense, changed to using this